app/controllers/api/v1x0/icons_controller.rb

```diff
@@ -30,6 +30,9 @@ def raw_icon
         send_data(image,
                   :type        => MimeMagic.by_magic(image).type,
                   :disposition => 'inline')
+      rescue ActiveRecord::RecordNotFound
+        Rails.logger.info("Icon not found for params: #{params.keys.select { |key| key.end_with?("_id") }}")
```

Comment:
The logger will still fill up with all these entries yet right? I wonder if we should do a `Rails.logger.debug`?